packages/backend/server/src/plugins/copilot/session.ts

```diff
@@ -262,6 +272,12 @@ export class ChatSessionService {
             },
           });
         }
+        await tx.aiSession.update({
+          where: { id: sessionId },
+          data: {
+            promptName: state.prompt.name,
+          },
+        });
```

Comment:
update prompt name in independent function but not in `setSession` like `async updatePrompt()`